integration/src/main/java/scripts/testscripts/DatasetOperations.java

```diff
@@ -67,6 +67,7 @@ public void setup(List<TestUserSpecification> testUsers) throws Exception {
   private void setupWorkspace(TestUserSpecification user) throws Exception {
     rawlsClient = new RawlsClient(server, user);
     workspaceSource = rawlsClient.createTestWorkspace();
+    rawlsClient.ingestData(workspaceSource);
```

Comment:
This change fixes a bug in the test code. Before it was ingesting data in both the source and destination workspaces, and the copy would fail since the destination workspace already had the same data as the source.